wesnoth-dm.po

```diff
@@ -4783,11 +4776,11 @@ msgid ""
 "Fire. That mission would cost Lionel his life; and neither Lionel nor "
 "Delfador knew that Lionel’s mission had been compromised from the start."
 msgstr ""
-"Po návratu do Weldynu Delfador zjistil, že se královna vydala s poměrně "
-"silným doprovodem na návštěvu svého rodného panství a svých nemocných "
-"rodičů. Lionel, králův nejdůvěryhodnější generál, byl na žádost Ašvíry "
-"poslán do Knalgy najít a získat Ohnivé žezlo. Tato mise bude stát Lionela "
-"život a ani Lionel ani Delfador nevěděli, že byla od samého začátku "
+"Po návratu do Weldynu Delfador zjistil, že Krelanova kniha zmizela. Královna "
+"odešla se silným doprovodem na návštěvu svého rodného panství a svých "
+"nemocných rodičů. Lionel, králův nejdůvěryhodnější generál, byl na žádost "
+"Ašvíry poslán do Knalgy najít a získat Ohnivé žezlo. Tato mise mohla stát "
```

Comment:
Lionel v HttT skutečně umírá, hledal jsem to, než jsem would přeložil jako bude - vizte scenarios/15_The_Lost_General
```suggestion
"Ašvíry poslán do Knalgy najít a získat Ohnivé žezlo. Tato mise bude stát "
```